### PR TITLE
Implement plant action for single user, tested

### PR DESCRIPTION
**Pull request by girikuncoro**
Merge #12

### CODE REVIEW

File: app/api/sms/plant_action.py [hunk 1]
```diff
@@ -1,14 +1,26 @@
...
         super(PlantAction, self).__init__(command)
 
+    def execute(self):
+        user = self.command.sms.user[0]
+        new = Farm(id=Farm.id(),
+                   district_id=user.district_id,
+                   action=self.CMD,
+                   crop_name=self.command.plant,
+                   quantity=self.command.amount)
+        new.put()
```
Comment (Ragil) [inline app/api/sms/plant_action.py]: Do each of these have an sms response to the user to indicate that it was successful?

File: app/api/sms/plant_action.py [hunk 2]
```diff
@@ -1,14 +1,26 @@
...
     """
     Execute a plant operation
     """
+    CMD = 'plant'
+
     def __init__(self, command):
         super(PlantAction, self).__init__(command)
 
+    def execute(self):
+        user = self.command.sms.user[0]
```
Comment (Ragil) [inline app/api/sms/plant_action.py]: The sms request should have 1 user. We should move this logic into the request layer at main.py

File: app/api/sms/plant_action.py (hunk 1 above)
Comment (girikuncoro) [inline app/api/sms/plant_action.py]: My mistake. Adding something like `return "plant command succeeded"` here should be fine right?

Comment (Ragil) [inline app/api/sms/plant_action.py]: internationalized :)

Comment (girikuncoro) [inline app/api/sms/plant_action.py]: `return _("plant command succeeded")` and add the definition in `i18n`?

Comment (Ragil) [inline app/api/sms/plant_action.py]: that should be enough. Convention wise, i think we should try to be grammatically correct with all string response? eg. start with capital letter.

**Comment by girikuncoro** [comment]
Fixed in #39

**Pull request merged**